mantidproject/mantid
# Fix ConvertToMD with workspace pointer input

ConvertToMD would give an error if the input workspace was not in the ADS, as it would not have a name. This change means the input workspace no longer needs a name.

**To test:**
```
ws = Load(SOME_WORKSPACE_PATH, StoreInADS=False)
ConvertToMD(InputWorkspace=ws, OutputWorkspace='output_workspace', QDimensions='|Q|',
                                 PreprocDetectorsWS='-', dEAnalysisMode='Direct')
```
Should run without errors.

Fixes #22347 

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
